client/jetpack-cloud/sections/comparison/table/useComparisonData.tsx

```diff
@@ -308,15 +312,7 @@ export const useComparisonData = () => {
 						name: translate( 'Ad network' ),
 						url: links.ad_network,
 						info: {
-							SECURITY: { content: <CheckIcon /> },
-							COMPLETE: { content: <CheckIcon /> },
-						},
-					},
-					{
-						id: 'google_analytics',
-						name: translate( 'Google Analytics integration' ),
-						url: links.google_analytics,
-						info: {
```

Comment:
Was this google analytics section meant to be removed for all products?

Reply:
Yes, sir.